data/explorations/classifier_demo_exploration.yaml

```diff
@@ -0,0 +1,76 @@
+author_notes: ''
```

Comment:
Probably. But changing the history at this point seems a lot of trouble. Is it ok if I just left it?